tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -0,0 +1,163 @@
+from robottelo.datafactory import gen_string, valid_data_list
+from robottelo.decorators import fixture, parametrize
+from robottelo.config import settings
+from nailgun import entities
+import pytest
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@parametrize('name', **valid_data_list('ui'))
+def test_positive_create_docker(session, name):
+    docker_url = settings.docker.external_url
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'Docker',
+            'provider_content.url': docker_url,
+        })
+        assert session.computeresource.search(name)[0]['Name'] == name
+
+
+@parametrize('name', **valid_data_list('ui'))
+def test_positive_create_ec2(session, name):
+    ec2_access_key = settings.ec2.access_key
+    ec2_secret_key = settings.ec2.secret_key
+    with session:
+        session.computeresource.create({
+            'name': name,
+            'description': gen_string('alpha'),
+            'provider': 'EC2',
```

Comment:
In [1]: from robottelo.constants import FOREMAN_PROVIDERS

In [2]: print FOREMAN_PROVIDERS['rhev']
RHV

In [3]: print FOREMAN_PROVIDERS['docker']
Docker

In [4]: print FOREMAN_PROVIDERS['ec2']
EC2